src/lumigo_tracer/spans_container.py

```diff
@@ -137,6 +152,9 @@ def handle_timeout(self, *args):
             get_logger().info("The tracer reached the end of the timeout timer")
             spans_id_copy = self.span_ids_to_send.copy()
             to_send = [self.spans[span_id] for span_id in spans_id_copy]
+            enrichment_span = self.generate_enrichment_span()
+            if enrichment_span:
+                to_send.append(enrichment_span)
```

Comment:
Do you think that maybe it will be better to put it at the beginning of `to_send` instead of the end?
In case there are a lot of http calls and we don't send all of them, we still want to get the `enrichment_span`.

Reply:
Interesting! that's a great idea!!